_config.yml

```diff
@@ -79,6 +79,8 @@ responsive_image:
     - width: 1400
   default_quality: 95
   strip: true
+  save_to_source: false
+  cache: true
```

Comment:
Can you point me to the documentation for these 2 configurations?

Reply:
Here's[https://github.com/wildlyinaccurate/jekyll-responsive-image#all-configuration-options](url) all of the available configuration options for the responsive images straight from the [Jekyll plugin official page](https://github.com/wildlyinaccurate/jekyll-responsive-image) that we're already using.

Reply:
Looks good overall, please add In the PR description, what is the time improvement from this change.

Reply:
Done